Lib/javascript/v8/javascriptcode.swg

```diff
@@ -334,9 +334,13 @@ fail:
 #else
     $jswrapper(args, errorHandler);
     if(errorHandler.err.IsEmpty()) {
-      return;
+      SWIGV8_RETURN(jsresult);
+    }
+    else {
+      SWIGV8_RETURN(SWIGV8_UNDEFINED());
```

Comment:
I think you don't want the `else` here, as it makes the addition below unreachable code in the case where this `#else` branch is used.